Update README.md, Add .tf files, .gitignore and diagram.png

renan-miranda opened the pull request:
Request to merge dev branch with master.

renan-miranda committed:
Update README.md, Add .tf files, .gitignore and diagram.png

diegows commented on README.md:
s/store/state/

diegows commented on README.md:
Editing the vars.tf file is not the best practice here. How can we do this better?

diegows commented on README.md:
We don't need base64, check file() or template() functions in Terraform documentation.

diegows reviewed (changes requested):
Indentation is not consistent, you are using 2 spaces and 4 spaces. Please check.

renan-miranda committed:
Fix identation, remove commented variables

renan-miranda committed:
Add 1 git-crypt collaborator

New collaborators:

	BB1FE65A Renan Miranda <renan.f.de.miranda@gmail.com>

renan-miranda committed:
Add .gitattributes and .gitignore to encrypt data and ignore the encryption key

renan-miranda committed:
Add resources dir

renan-miranda committed:
Remove variables from bucket_s3

Now the creation of S3 bucket for remote state
will be based on a random id instead of using variables.
After, the bucket name will be stored in resources/remote_s3_bucket.sh
to be used in main execution.

renan-miranda committed:
Change on .gitattributes to encrypt specific files

renan-miranda committed:
Changes on main directory

- user_data stored in resources/user_data.txt instead of base64 variable
- addresses.txt changed from ./main to ./resources
- Added resources/main.tfvars for execution

renan-miranda committed:
Add .gitignore on resources directory

renan-miranda committed:
Update on README.md

renan-miranda committed:
Remove user_base64 from resources/main.tfvars

renan-miranda committed:
Minor update on README.md

renan-miranda committed:
Minor update on README.md

diegows commented on README.md:
Why are you encrypting this file?

diegows commented on README.md:
Some spanish in that phrase :)

diegows reviewed (changes requested):
Could you add an example how to use git-crypt?
